### ISSUE TITLE
How to do grouping inside nested range?

### ISSUE THREAD

**Issue by fingers10** [ClosedXML/ClosedXML.Report#102]
Hi 

Grouping works inside nested range? Any sample template for that? Please assist.

Thanks,
Abdul

**Comment by b0bi79** [comment]
Send what you want to get. I’ll think about how to help.

**Comment by fingers10** [comment]
Example in the attached excel, Category values is placed in both column B and C
[NestedGrouping Test.xlsx](https://github.com/ClosedXML/ClosedXML.Report/files/3537464/NestedGrouping.Test.xlsx)

**Comment by fingers10** [comment]
Any ideas/hints for this?

**Comment by b0bi79** [comment]
Sorry for the long reply. I studied your template and thought for a long time how to make it work. We can add support for groups in nested regions, but this is difficult to implement. It takes a lot of time. I will leave this task for the future.

**Comment by fingers10** [comment]
Yeah No problem. Thanks for the reply. Right now the workaround is to do necessary grouping using LINQ in code and we can get the results as per our requirements. I'm happy that I can atleast contribute by finding issues.

Cheers,
Abdul
